homesensorhub/sensors/environmental_sensor.py

```diff
@@ -39,11 +40,26 @@ def collect_data(self) -> dict:
 
         :return: None
         """
+        sensor = self.get_sensor()
+
         sensor_data = {
-                     'temperature': self.get_sensor().temperature,
-                     'humidity': self.get_sensor().humidity,
-                     'pressure': self.get_sensor().pressure,
-                     'altitude': self.get_sensor().altitude
+            'temperature': self.build_sensor_packet('temperature',
+                                                    sensor.temperature,
+                                                    datetime.datetime.now(),
+                                                    'celsius'
+                                                    ),
+            'humidity': self.build_sensor_packet('humidity',
+                                                 sensor.humidity,
+                                                 datetime.datetime.now(),
+                                                 'percent'),
+            'pressure': self.build_sensor_packet('pressure',
+                                                 sensor.pressure,
+                                                 datetime.datetime.now(),
+                                                 'UF/USA'),
```

Comment:
what does UF/USA mean ?

Reply:
Units of Force / Units of squared Area. It was supposed to be the measurement for the pressure, but after checking the adafruit bme280 library, it is measured in hectoPascals (I will change it):

 @property
    def pressure(self):
        """
        The compensated pressure in hectoPascals.
        returns None if pressure measurement is disabled
        """
https://github.com/adafruit/Adafruit_CircuitPython_BME280/blob/master/adafruit_bme280.py